web/sites/default/config/rabbit_hole.behavior_settings.node_type_page.yml

```diff
@@ -0,0 +1,11 @@
+uuid: 2e904430-77e2-4303-a7ff-77f3807df626
+langcode: en
+status: true
+dependencies:
+  config:
+    - node.type.page
+id: node_type_page
+action: display_page
+allow_override: 1
```

Comment:
@radmosley, I'm not sure why this was in the config export. Did you touch rabbit hole settings at all?